compatibility/sandbox-migration/SandboxMigrationRunner.hs

```diff
@@ -0,0 +1,181 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+module Main (main) where
+
+-- This test runs through the following steps:
+-- 1. Start postgres
+-- 2. Start the oldest version of sandbox.
+-- 3. Upload a DAR using the same SDK version.
+-- 4. Stop sandbox.
+-- 5. In a loop over all versions:
+--    1. Start sandbox of the given version.
+--    1. Run a script for querying and creating new contracts.
+--    2. Stop sandbox.
```

Comment:
```suggestion
--    2. Run a script for querying and creating new contracts.
--    3. Stop sandbox.
```

Reply:
Counting to 3 seems to be too hard for me :facepalm: